### ISSUE TITLE
type not triggering onChange callback in ReactJS app

### ISSUE THREAD

**Issue by vincentaudebert** [segmentio/nightmare#381]
Hi guys,

Just having an issue with `type` function.
In my code I call 
`yield night.type('div.card__body > input.field-metric', '50000')` 
which is meant to trigger a onChange callback and update some value in an object.
It works perfectly when I fill manually my input on Chrome but when I run tests with nightmarejs value is never updated.

We found a temp fix which is 
```
nightmare.action('type', function(selector, text, done) {
    this.evaluate_now(function(selector, text) {
        const elem = document.querySelector(selector);
        elem.focus();
        elem.value = text;
        elem.blur();
        elem.dispatchEvent(new Event('input', { bubbles: true }));
    }, done, selector, text);
});
```
But we thought it might be a issue you want to have a look into on your side.

Cheers.

**Comment by mokevnin** [comment]
+1

**Comment by matthewmueller** [comment]
okay this should be fixed now in `2.1.0`, let me know if it's still broken!
